src/Tests/Helpers/Extensions.cs

```diff
@@ -0,0 +1,69 @@
+namespace SFA.DAS.Funding.SystemAcceptanceTests.Helpers;
+
+public static class Extensions
+{
+    public static void ShouldHaveCorrectFundingPeriods(this List<DeliveryPeriod> periods, int numberOfInstalments, short firstDeliveryPeriod, short firstDeliveryAcademicYear)
+    {
+        if (firstDeliveryPeriod > 12) throw new ArgumentException("Period can't be greater than 12 can it?", nameof(firstDeliveryPeriod));
```

Comment:
It can't yes, R13 and 14 are collection periods only and this is referring to delivery periods.

Reply:
so is the logic not right?

Reply:
no it's correct I was just confirming since it looks like a question in the code :D